client/profile-wizard/steps/usage-modal.js

```diff
@@ -66,8 +59,7 @@ class UsageModal extends Component {
 		}
 	}
 
-	updateTracking() {
-		const { allowTracking } = this.state;
+	updateTracking( { allowTracking } ) {
```

Comment:
I'm wondering if we should add tracks here to see when the tracking is accepted and record the title of the modal so we know where it was turned on.  Maybe we already have this information through another event though.  /cc @timmyc @pmcpinto

Reply:
We can do this, but don't think it'll help very much. Technically any plugin can enable whole site tracking by setting option to true. On the other hand we can record tracking enabled event in callback of `wcTracks.enable`.

Reply:
I was thinking more along the lines of so we could determine which modal (for split testing purposes) provided more opt-ins.

But I suppose we could just put a filter on the option server-side and record the referrer if we're interested in this info since multiple usage modal styles/text don't currently appear on the same page.